lms/templates/ccx/coach_dashboard.html

```diff
@@ -167,7 +167,7 @@
   function validateForm(form) {
     var newCCXName = $(form).find('#ccx_name').val();
     var $errorMessage =  $('#ccx-create-message');
-    var hasCcxConnector = ${has_ccx_connector | n, dump_js_escaped_json};
+    var hasCcxConnector = ${has_ccx_connector};
```

Comment:
You should back out this change-- what was here before was correct.

http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#mako-template-files